docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -460,53 +460,55 @@ You can set the following YBA Installer configuration options.
 | `server_cert_path`<br />`server_key_path` | If providing custom certificates, give the path with these values. If not provided, the installation process generates self-signed certificates. Optional. |
 | `service_username` | The Linux user that will run the YBA processes. Default is `yugabyte`. The install process will create the `yugabyte` user. If you wish to use a different user, create that user beforehand and specify it in `service_username`. YBA Installer only creates the `yugabyte` user, not custom usernames. |
 
+Note that you can't change the `installRoot` and `service_username` settings after installation.
```

Comment:
Can' we star them in the table or something and then the star points down here? It's kind of hidden and might not be as clear.